cowprotocol/competitor_analysis/dex_and_aggregator_transactions_by_sender_layered_4836358.sql

```diff
@@ -0,0 +1,98 @@
+-- Calculates transactions USD volume by sender, where only top-level trade is kept.
+-- Takes into account both dex and dex-aggregators.
+--
+-- Parameters:
+--  {{start_time}} - the trade timestamp for which the analysis should start (inclusive)
+--  {{end_time}} - the trade timestamp for which the analysis should end (inclusive)
+
+with dex_aggregator_trades as (
+    select
+        da_trades.tx_from,
+        da_trades.tx_hash,
+        da_trades.blockchain,
+        da_trades.project,
+        sum(da_trades.amount_usd) as amount_usd,
+        any_value(da_trades.block_time) as block_time,
+        case
+            -- Priorities are chosen in a way that this measures which interface the volume was traded from.
+            when da_trades.project = 'cow_protocol' then 1 -- GP aggregates the aggregators
```

Comment:
Can you check how 1Inch Fusion, Paraswap Delta and Uniswap X are represented and if they also need to be considered in the layering here? This query was likely written before they existed.

Like, looking at this now, I think it should be all intent protocols on priority 1 and any other aggregator on priority 2 and then any dex liquidity on priority 3